docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -0,0 +1,331 @@
+1. Add the following configuration to the file:
+   ```
+   nginx:
+     ingress:
+       enabled: true
+       ingressClassName: nginx
+       hosts:
+         - host: <ingress-host>
+           paths:
+             - path: /
+               pathType: Prefix
+       tls:
+         # empty, disabled.
+   ```
+
+
+## Install Grafana Tempo using the Helm chart
+
+Use the following command to install Tempo using the configuration options you’ve specified in the `custom.yaml` file:
+
+```bash
+helm -n tempo-test install tempo grafana/tempo-distributed -f custom.yaml
+```
+
+>**NOTE**: The output of the command contains the write and read URLs necessary for the following steps.
+
+If the installation is successful, the output will be similar to this:
+
+```yaml
+>  helm -n tempo-test install tempo grafana/tempo-distributed -f custom.yaml
+
+W0210 15:02:09.901064    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+W0210 15:02:09.904082    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+W0210 15:02:09.906932    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+W0210 15:02:09.929946    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+W0210 15:02:09.930379    8613 warnings.go:70] spec.template.spec.topologySpreadConstraints[0].topologyKey: failure-domain.beta.kubernetes.io/zone is deprecated since v1.17; use "topology.kubernetes.io/zone" instead
+NAME: tempo
+LAST DEPLOYED: Fri Feb 10 15:02:08 2023
+NAMESPACE: tempo-test
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+***********************************************************************
+ Welcome to Grafana Tempo
+ Chart version: 1.0.1
+ Tempo version: 2.0.0
+***********************************************************************
+
+Installed components:
+* ingester
+* distributor
+* querier
+* query-frontend
+* compactor
+* memcached
+```
+
+>**NOTE**: If you update your `values.yaml` or `custom.yaml`, run the same helm install command and replace `install` with `upgrade`.
+
+Check the statuses of the Tempo pods:
+
+```bash
+kubectl -n tempo-test get pods
+```
+
+The results look similar to this:
+
+```
+NAME                                    READY   STATUS    RESTARTS   AGE
+tempo-compactor-86cd974cf-8qrk2         1/1     Running   0          22h
+tempo-distributor-bbf4889db-v8l8r       1/1     Running   0          22h
+tempo-gateway-75f468bd46-fbzj4          1/1     Running   0          22h
```

Comment:
Delete this line to be consistent with the values we provided.  This was a carry-over from a previous `helm install` on my part I think.

Reply:
Done.